Extension/UqlFunctionCaller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Extension;
+
+use Netdudes\DataSourceryBundle\Extension\Exception\FunctionNotFoundException;
+
+class UqlFunctionCaller
```

Comment:
We created this class to split the logic a bit into smaller classes while refactoring. Mainly because extension container class didn't look like a good place to call a UQL function (it should manage it's collection of extensions).
Also, previously this class was bigger, with a logic of handling `ContextAwareUqlFunction` / `UqlFunction` objects.
But you're right, now this class is small. We could think about moving this logic somewhere else.